source/common/midi_processing.py

```diff
@@ -0,0 +1,174 @@
+import dawdreamer as daw
+from scipy.io import wavfile
+from pydub import AudioSegment
+import os
+
+sample_rate = 44100
+buffer_size = 128
+render_time = 4
+model_midi_path = os.path.join(os.getcwd(), 'files', 'audio_generator', 'midi_files')
+model_preset_path = os.path.join(os.getcwd(), 'files', 'audio_generator', 'preset_files')
+model_wav_path = os.path.join(os.getcwd(), 'files', 'audio_generator', 'wav_files')
+model_plugin_path = os.path.join(os.getcwd(), 'files', 'audio_generator', 'StupidSimpleSampler')      
+
+def drum(list_of_objects, amount_of_drum):
+    for shape in list_of_objects:
+        if amount_of_drum == 0:
+            engine = daw.RenderEngine(sample_rate, buffer_size)
+            synth = engine.make_plugin_processor("my_synth", model_plugin_path + "\\StupidSimpleSampler.dll")
+            assert synth.get_name() == "my_synth"
+            synth.load_state(model_preset_path + "\\kick.fxb")
+            #synth.open_editor()
+            synth.load_midi(model_midi_path + "\\drum_output.mid", clear_previous=False, beats=False, all_events=False) 
+            engine.load_graph([
+                            (synth,[])
+            ])
+            engine.set_bpm(shape.bpm)
+            engine.render(render_time)  
+            audio = engine.get_audio()  
+            wavfile.write(model_wav_path + "\\drum.wav",  sample_rate, audio.transpose()) 
+            amount_of_drum +=1
+
+def guitar(list_of_objects, amount_of_guitar):      
+    for shape in list_of_objects:  
+        if amount_of_guitar == 0:
+            engine = daw.RenderEngine(sample_rate, buffer_size)
+            synth = engine.make_plugin_processor("my_synth", model_plugin_path + "\\StupidSimpleSampler.dll")
+            assert synth.get_name() == "my_synth"
+            synth.load_state(model_preset_path + "\\guitar.fxb")
```

Comment:
just check the whole document for \\

Reply:
@Maucee,
No, This will be addressed in the https://github.com/Holandsoest/art-to-music/issues/19